automatic/php/tools/chocolateyInstall.ps1

```diff
@@ -7,32 +7,38 @@ $installLocation = GetInstallLocation "$toolsPath\.."
 
 if ($installLocation) {
   Write-Host "Uninstalling previous version of php..."
-  UninstallPackage -libDirectory "$toolsPath\.." -packageName 'php'
+  UninstallPackage -libDirectory "$toolsPath\.." -packageName $env:ChocolateyPackageName
   Uninstall-ChocolateyPath $installLocation
 }
 
 $pp = Get-PackageParameters
 
-$downloadInfo = GetDownloadInfo -downloadInfoFile "$toolsPath\downloadInfo.csv" -parameters $pp
+$downloadInfo = @{
```

Comment:
`downloadInfo` is probably not the best to be used for this variable anymore.
Maybe `filesInfo`?